src/m365/spo/commands/listitem/listitem-get.spec.ts

```diff
@@ -191,15 +191,66 @@ describe(commands.LISTITEM_GET, () => {
     });
   });
 
+  it('returns listItemInstance object when list item is requested with an output type of json, a list of fields with lookup field are specified', (done) => {
+    sinon.stub(request, 'get').callsFake((opts: any) => {
+      if ((opts.url as string).indexOf('&$expand=') > -1) {
+        actualId = parseInt(opts.url.match(/\/items\((\d+)\)/i)[1]);
+        return Promise.resolve(
+          {
+            "Attachments": false,
+            "AuthorId": 3,
+            "ContentTypeId": "0x0100B21BD271A810EE488B570BE49963EA34",
+            "Created": "2018-03-15T10:43:10Z",
+            "EditorId": 3,
+            "GUID": "ea093c7b-8ae6-4400-8b75-e2d01154dffc",
+            "ID": actualId,
+            "Modified": "2018-03-15T10:43:10Z",
+            "Title": expectedTitle,
+            "Company": `{ "Title": "Contoso" }`
+          }
+        );
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    command.allowUnknownOptions();
+
+    const options: any = {
+      debug: false,
+      listTitle: 'Demo List',
+      webUrl: 'https://contoso.sharepoint.com/sites/project-x',
+      id: expectedId,
+      output: "json",
+      properties: "ID,Modified,Company/Title"
+    };
+
+    command.action(logger, { options: options } as any, () => {
+      try {
+        assert({
```

Comment:
This will always return true. We should do a deep strict equal instead and compare it to the value returned by the API